test/spec/modules/emx_digitalBidAdapter_spec.js

```diff
@@ -4,167 +4,241 @@ import * as utils from 'src/utils';
 import { newBidder } from 'src/adapters/bidderFactory';
 
 describe('emx_digital Adapter', function () {
-  const adapter = newBidder(spec);
-
-  describe('required function', function () {
+  describe('callBids', function () {
+    const adapter = newBidder(spec);
     it('exists and is a function', function () {
       expect(adapter.callBids).to.exist.and.to.be.a('function');
     });
   });
 
   describe('isBidRequestValid', function () {
-    let bid = {
-      'bidder': 'emx_digital',
-      'params': {
-        'tagid': '25251'
-      },
-      'mediaTypes': {
-        'banner': {
-          'sizes': [
-            [300, 250],
-            [300, 600]
-          ]
-        }
-      },
-      'adUnitCode': 'adunit-code',
-      'sizes': [
-        [300, 250],
-        [300, 600]
-      ],
-      'bidId': '30b31c2501de1e',
-      'bidderRequestId': '22edbae3120bf6',
-      'auctionId': '1d1a01234a475'
-    };
+    describe('banner request validity', function () {
+      let bid = {
+        'bidder': 'emx_digital',
+        'params': {
+          'tagid': '25251'
+        },
+        'mediaTypes': {
+          'banner': {
+            'sizes': [[300, 250]]
+          }
+        },
+        'adUnitCode': 'adunit-code',
+        'sizes': [
+          [300, 250],
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
+      let badBid = {
+        'bidder': 'emx_digital',
+        'params': {
+          'tagid': '25251'
+        },
+        'mediaTypes': {
+          'banner': {
+          }
+        },
+        'adUnitCode': 'adunit-code',
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
+      let noBid = {};
+      let otherBid = {
+        'bidder': 'emxdigital',
+        'params': {
+          'tagid': '25251'
+        },
+        'mediaTypes': {
+          'banner': {
+            'sizes': [[300, 250]]
+          }
+        },
+        'adUnitCode': 'adunit-code',
+        'sizes': [
+          [300, 250],
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
+      let noMediaSizeBid = {
+        'bidder': 'emxdigital',
+        'params': {
+          'tagid': '25251'
+        },
+        'mediaTypes': {
+          'banner': {}
+        },
+        'adUnitCode': 'adunit-code',
+        'sizes': [
+          [300, 250],
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
 
-    it('should return true when required params found', function () {
-      expect(spec.isBidRequestValid(bid)).to.equal(true);
+      // it('should return true when required params found', function () {
```

Comment:
@EMXDigital It seems there were some other commented unit tests.  Can you remove these as well?

Reply:
Apologies for missing this, it is also removed.